tools/nemo_forced_aligner/README.md

```diff
@@ -0,0 +1,82 @@
+
+> Note: NFA does not require `"duration"` fields in the manifest, and can align long audio files without running out of memory. Depending on your machine specs, you can align audios up to 5-10 minutes on Conformer CTC models, up to around 1.5 hours for QuartzNet models, and up to several hours for Citrinet models. NFA will also produce better alignments the more accurate the ground-truth `"text"` is.
+
+
+# Output CTM file format
+For each utterance specified in a line of `manifest_filepath`, several CTM files will be generated:
+* a CTM file containing token-level alignments at `<output_dir>/tokens/<utt_id>.ctm`,
+* a CTM file containing word-level alignments at `<output_dir>/words/<utt_id>.ctm`,
+* if `additional_ctm_grouping_separator` is specified, there will also be a CTM file containing those segments at `output_dir/additional_segments`.
+Each CTM file will contain lines of the format:
+`<utt_id> 1 <start time in samples> <duration in samples> <text, ie token/word/segment>`.
+Note the second item in the line (the 'channel ID', which is required by the CTM file format) is always 1, as NFA operates on single channel audio.
+
+# Output JSON manifest file format
+A new manifest file will be saved at `<output_dir>/<original manifest file name>_with_ctm_paths.json`. It will contain the same fields as the original manifest, and additionally:
+* `"token_level_ctm_filepath"`
+* `"word_level_ctm_filepath"`
+* `"additonal_segment_level_ctm_filepath"` (if `additional_ctm_grouping_separator` is specified)
+* `"pred_text"` (if `align_using_pred_text=true`)
+
+
+# How do I evaluate the alignment accuracy?
+Ideally you would have some 'true' CTM files to compare with your generated CTM files. 
+
+Alternatively (or additionally), you can visualize the quality of alignments using tools such as Gecko, which can play your audio file and display the predicted alignments at the same time. The Gecko tool requires you to upload an audio file and at least one CTM file. The Gecko tool can be accessed here: https://gong-io.github.io/gecko/. More information about the Gecko tool can be found on its Github page here: https://github.com/gong-io/gecko. 
+
+Note that Gecko may not display some tokens/words/segments properly if their timestamps are too short. To ameliorate this, you may try setting `minimum_timestamp_duration` to some suitable number. If you are analyzing token-level CTMs, it may also be helpful to set `remove_blank_tokens_from_ctm` to `True`, to make the Gecko visualization less cluttered.
```

Comment:
Add a bold over note, and maybe provide the flags in a list to 1) remove blanks 2) elongate token lengths.
Plus a note on how to perform comparison between alignments might be useful (but extra info, not really needed to be added)

Reply:
Thanks, done